config/initializers/devise.rb

```diff
@@ -17,10 +17,10 @@
   config.secret_key = Rails.application.secrets.secret_key_base
 
   # Configure the class responsible to send e-mails.
-  # config.mailer = "UserMailer"
+  config.mailer = "UserMailer"
```

Comment:
Cool! Is that why we name a method in the mailer `confirmation_instructions`? Does it need to match what devise is expecting?

Reply:
Yes, it has to be confirmation_instructions